cluster-scope/overlays/nerc-ocp-infra/machineconfigs/configure-bond0/src/ctl-0-nic1.nmconnection

```diff
@@ -0,0 +1,10 @@
+[connection]
+id=ens6f0
+type=ethernet
+interface-name=ens6f0
+master=bond0
```

Comment:
In the ethernet section there is no MAC address specified, as there is in the examples at: [https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_networking/assembly_manually-creating-networkmanager-profiles-in-key-file-format_configuring-and-managing-networking](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_networking/assembly_manually-creating-networkmanager-profiles-in-key-file-format_configuring-and-managing-networking). Is this omission possible because we are specifying the bond0 profile here which is created above?

Reply:
I think in those examples the MAC address is being used to identify the interface. In this case, we're using the `interface-name` parameter. It wouldn't make sense to specify a MAC address here because this profile is expected to apply on multiple machines.